src/Particles/solidParticle.ts

```diff
@@ -241,13 +246,13 @@ export class SolidParticle {
      * @returns true if it intersects
      */
     public intersectsMesh(target: Mesh | SolidParticle): boolean {
-        if (!this._boundingInfo || !target._boundingInfo) {
+        if (!this._boundingInfo || !target.getBoundingInfo()) {

```

Comment:
Should be:
```javascript
if (!this._boundingInfo || !target.hasBoundingInfo) {
```
instead, as `getBoundingInfo()` can't return `null` (I guess you will need to add a `hasBoundingInfo` to `SolidParticle`, however).

Reply:
I can do that